src/foraging_gui/MyFunctions.py

```diff
@@ -308,8 +308,9 @@ def _get_warmup_state(self):
             # turn on the warm up
             warmup=1
         # show current metrics of the warm up
-        self.win.WarmupWarning.setText('Finish trial: '+str(round(finish_trial,2))+ '; Finish ratio: '+str(round(finish_ratio,2))+'; Choice ratio bias: '+str(round(abs(choice_ratio-0.5),2)))
```

Comment:
I would check where this warning is currently located. I'm not sure if we want to add it to the warning queue

Reply:
Currently located in warning box: 
![image](https://github.com/user-attachments/assets/1a1f908a-bfb8-403a-bacf-2f4638a8883f)